mappings/net/minecraft/client/render/block/entity/ChestBlockEntityRenderer.mapping

```diff
@@ -7,3 +13,12 @@ CLASS net/minecraft/class_826 net/minecraft/client/render/block/entity/ChestBloc
 	FIELD field_4364 NORMAL_TEX Lnet/minecraft/class_2960;
 	FIELD field_4365 isChristmas Z
 	FIELD field_4366 ENDER_TEX Lnet/minecraft/class_2960;
+	METHOD method_22749 renderModel (Lnet/minecraft/class_287;Lnet/minecraft/class_630;Lnet/minecraft/class_630;Lnet/minecraft/class_630;FIILnet/minecraft/class_1058;)V
```

Comment:
Should I name this bufferModel as it's actually adding the model to the bufferbuilder